test/shallow_ice_test.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2017-2019 by Daniel Shapero <shapero@uw.edu>
+#
+# This file is part of icepack.
+#
+# icepack is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# The full text of the license can be found in the file LICENSE in the
+# icepack source directory or at <http://www.gnu.org/licenses/>.
+
+import numpy as np
+
+# Test our numerical solvers against this analytical solution.
+import pygmsh
+import os
+import matplotlib.pyplot as plt
+import firedrake
+from firedrake import inner, grad, dx, interpolate, as_vector, max_value
+import icepack, icepack.models
+from icepack.constants import (ice_density as ρ_I, water_density as ρ_W,
```

Comment:
I think the water density and the Weertman sliding law imports can be removed for now.